content/about/contribute/writing-a-new-topic.md

```diff
@@ -294,18 +294,44 @@ This will be rendered as:
 $ istioctl create -f @samples/bookinfo/kube/route-rule-reviews-v3.yaml@
 ```
 
+## Displaying file snippets
+
+It is often useful to display portions of a larger file. You can annotate a text file to create named snippets within the file by
+using the `$snippet` and `$endsnippet` annotations. For example, you could have a text file that looks like this:
+
+{{< file_content file="examples/snippet_example.txt" lang="plain" >}}
```

Comment:
This gets into the weeds of the file formats. May be its easier to start with just the YAML and say people have to add comments to YAMLs with 
```
# $startsnippet Name
some YAML
# $endsnippet Name
```